tests/src/JIT/Methodical/tailcall_v4/hijacking.ilproj

```diff
@@ -14,6 +14,10 @@
     <ReferencePath>$(ProgramFiles)\Common Files\microsoft shared\VSTT\11.0\UITestExtensionPackages</ReferencePath>
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\</SolutionDir>
     <NuGetPackageImportStamp>7a9bfb7d</NuGetPackageImportStamp>
+
+    <!-- NOTE: this test simply takes too long to complete under heap verify. It is not fundamentally incompatible. -->
```

Comment:
Not sure if you want to use the "GC stress/heap verify" version of the comment on these.